### PR TITLE
feat: add tooltips when adding and removing from nft bag

### PR DESCRIPTION
**Pull request by lynnshaoyu**
THANK YOU TO @JackShort for making the tooltip behavior when scrolling less glitchy with this PR: https://github.com/Uniswap/interface/pull/5238 . including his changes in this merge. 

https://uniswaplabs.atlassian.net/browse/WEB-2073?atlOrigin=eyJpIjoiNzFhMzVlNGUzYjZhNDdjNTg3ZTBhZGE1YTUxNTdiNjEiLCJwIjoiaiJ9

### CODE REVIEW

File: src/nft/components/collection/CollectionAsset.tsx [hunk 1]
```diff
@@ -71,6 +87,21 @@ export const CollectionAsset = ({
     }
   }, [addAssetsToBag, asset, bagExpanded, bagManuallyClosed, isMobile, setBagExpanded, trace])
 
+  useEffect(() => {
+    if (isSelected !== prevIsSelected && !inSweep) {
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.tsx:91]: The sweep condition doesn't work exactly as when I manually add items while the modal is open I don't get the memo. In addition, after I've closed the sweep modal, the tooltips don't show.

File: src/nft/components/collection/CollectionAsset.tsx [hunk 2]
```diff
@@ -82,24 +113,40 @@ export const CollectionAsset = ({
...
-        ) : assetMediaType === AssetMediaType.Video ? (
-          <Card.Video shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        ) : (
-          <Card.Audio shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        )}
-      </Card.ImageContainer>
+      <Tooltip
+        text={
+          <Box as="span" className={bodySmall} style={{ color: themeVars.colors.textPrimary }}>
+            <Trans>{isSelected ? ADDED_TO_BAG_TOOLTIP_TEXT : REMOVED_FROM_BAG_TOOLTIP_TEXT}</Trans>{' '}
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.tsx:119]: My understanding of the `Trans` component is that it needs to wrap the exact string and can't wrap conditionals, did I have that wrong?

File: src/nft/components/collection/CollectionAsset.tsx [hunk 3]
```diff
@@ -82,24 +113,40 @@ export const CollectionAsset = ({
...
-          <Card.Image />
-        ) : assetMediaType === AssetMediaType.Video ? (
-          <Card.Video shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        ) : (
-          <Card.Audio shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        )}
-      </Card.ImageContainer>
+      <Tooltip
+        text={
+          <Box as="span" className={bodySmall} style={{ color: themeVars.colors.textPrimary }}>
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.tsx:118]: you can just do `color: 'textPrimary'` or do a styled component

File: src/nft/components/profile/view/ViewMyNftsAsset.tsx [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
 import { themeVars } from 'nft/css/sprinkles.css'
 import { useBag, useIsMobile, useSellAsset } from 'nft/hooks'
 import { TokenType, WalletAsset } from 'nft/types'
+import { useEffect, useState } from 'react'
 import { useMemo } from 'react'
 import { useNavigate } from 'react-router-dom'
+
+const TOOLTIP_TIMEOUT = 2000
+const ADDED_TO_BAG_TOOLTIP_TEXT = 'Selected'
+const REMOVED_FROM_BAG_TOOLTIP_TEXT = 'Deselected'
```
Comment (cbachmeier) [inline src/nft/components/profile/view/ViewMyNftsAsset.tsx:17]: nit: rename this to SELECT and DESELECT_TOOLTIP_TEXT

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/nft/components/collection/CollectionAsset.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/CollectionAsset.tsx:91]: u r right it was inconsistent. my changes now should have it working consistently.

File: src/nft/components/profile/view/ViewMyNftsAsset.tsx (hunk 4 above)
Comment (lynnshaoyu) [reply src/nft/components/profile/view/ViewMyNftsAsset.tsx:17]: done

File: src/nft/components/collection/CollectionAsset.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/nft/components/collection/CollectionAsset.tsx:119]: i'm not sure, but changed it to be safe

**Review by cbachmeier — APPROVED** [review]
LGTM

**Review by JackShort — CHANGES_REQUESTED** [review]
changes requested over slack

**Pull request merged**